test/unit/sourceset.test.js

```diff
@@ -0,0 +1,479 @@
+/* eslint-env qunit */
+import videojs from '../../src/js/video.js';
+import document from 'global/document';
+import window from 'global/window';
+import log from '../../src/js/utils/log.js';
+import sinon from 'sinon';
+
+const Html5 = videojs.getTech('Html5');
+const wait = 1;
```

Comment:
@BrandonOCasey do you remember why this was set to 100 initially? Setting it to 1 halved the testing time for this module.

Reply:
one of the IE versions was too slow, and tests would time out and fail

Reply:
well, everything seems to have passed on the first run. We can always set it to a slightly larger number later on for IEs or something.